src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -1425,6 +1426,12 @@ def aks_operation_abort(cmd,   # pylint: disable=unused-argument
     return sdk_no_wait(no_wait, client.begin_abort_latest_operation, resource_group_name, name, headers=headers)
 
 
+def aks_machine_list(cmd, client, resource_group_name, cluster_name, agentpool_name):
```

Comment:
Could you please add a live test case for these 2 new commands? Could find samples from `test_aks_commands.py`